.github/workflows/deploy_reusable.yml

```diff
@@ -116,6 +116,12 @@ jobs:
         slot-name: production
         images: '${{ inputs.REGISTRY }}/${{ inputs.REPO }}:${{ github.sha }}'
 
+    - name: Azure Swap Slots
+      uses: azure/CLI@v2
+      with:
+        inlineScript: |
+          az webapp deployment slot swap -n ${{ inputs.APP }} --slot pre-live --resource-group csels-rsti-${{ inputs.ENVIRONMENT }}${{ github.event.number }}-moderate-rg
```

Comment:
Do we want auto swap here? Otherwise I believe we need to do --target-slot(?)

Reply:
`--target-slot` defaults to `production` when not specified, and `auto swap` unfortunately doesn't work for linux